dashboard/app/models/blockly.rb

```diff
@@ -270,23 +287,9 @@ def blockly_options
         level_prop.delete('fn_failureCondition')
       end
 
-      app_options = {}
-
-      app_options[:levelGameName] = level.game.name if level.game
-      app_options[:skinId] = level.skin if level.is_a?(Blockly)
-
       # Set some values that Blockly expects on the root of its options string
-      non_nil_level_prop = level_prop.reject!{|_, value| value.nil?}
-      app_options.merge!({
-                             baseUrl: Blockly.base_url,
-                             app: level.game.try(:app),
-                             levelId: level.level_num,
-                             level: non_nil_level_prop,
```

Comment:
`:level` is being set in levels_helper.rb.

I think I misread https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/appMain.js#L29 as making `levelId` unnecessary, but that doesn't seem to be the case. I just added `:levelId` alongside `:level` in levels_helper.rb, but I'll need to move it into `blockly_level_options` and refactor the JS a a bit (in another PR).